src/css/global/_typography.scss

```diff
@@ -27,64 +27,64 @@
 /* Typography mixins */
 @mixin text--header-1 {
     @include bold;
-    font-size: 2.5rem;
+    font-size: 2.5rem !important;
```

Comment:
Why do you need to use `!important`? I thought it was a sin to use it :rofl:

Reply:
hah not always. in this case it is because we need it to override quasar defaults unfortunately. we also use it in utility classes in the same file as it is common practice to ensure utility class styles (which should only be 1 attribute each) take effect